app/models/tutorial.rb

```diff
@@ -1,5 +1,9 @@
 class Tutorial < ApplicationRecord
-  validates :title, presence: true
+  include FriendlyId
+
+  friendly_id :title
 
   has_many :tutorial_steps
+
+  validates :title, presence: true
```

Comment:
There's no defined organization for all the stuff that can end up in a model. Everyone's got their style. I did some digging around this week and (for myself at least…) came up with:

1. Mixins (e.g. `include SomeModule`)
1. Miscellaneous stuff (constants, `friendly_id :foo`, etc.)
1. Associations (`belongs_to`, `has_many`, etc.)
1. Validations
1. Scopes
1. Callbacks (`before_save`, `after_create`, etc.)
1. Instance methods
1. Class methods
1. Private methods

🤷‍♂️